hack/update-vendor.sh

```diff
@@ -431,3 +431,6 @@ reviewers:
 __EOF__
 
 kube::log::status "NOTE: don't forget to handle vendor/* and LICENSE/* files that were added or removed" >&11
+
+# TODO simplify update-vendor and eliminate this
+hack/update-go-workspaces.sh
```

Comment:
nit, move this above the final output and log it is happening (just like we log and call out to update-vendor-licenses)